src/overlays/actors/ovl_Obj_Moon_Stone/z_obj_moon_stone.c

```diff
@@ -1,5 +1,5 @@
 /*
- * File z_obj_moon_stone.c
+ * File: z_obj_moon_stone.c
  * Overlay: ovl_Obj_Moon_Stone
  * Description: Moon's Tear
```

Comment:
If only we could work in a reference to https://en.wikipedia.org/wiki/The_Moonstone ... ah well.